hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetQuotaRequest.java

```diff
@@ -228,11 +227,24 @@ public boolean checkQuotaBytesValid(OMMetadataManager metadataManager,
     return true;
   }
 
-  public boolean checkQuotaNamespaceValid(long quotaInNamespace) {
-
+  public boolean checkQuotaNamespaceValid(OMMetadataManager metadataManager,
```

Comment:
Similar to this, I assume we already have proper checks for bucket namespace quota is enabled, the total number of keys under the bucket can not be updated in case bucket already has more number of keys.

Reply:
Yes, it’sproper